### PR TITLE
260 disabled org name edition if non admin

### PR DESCRIPTION
**Pull request by MLE-Astek**

### CODE REVIEW

File: portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/create-or-modify-organization.jsx.js [hunk 1]
```diff
@@ -50,7 +49,9 @@ var CreateOrModifyOrganizationModal = React.createClass({
                                                         organization={this.state.organization}
                                                         step={this.state.step}
                                                         onStepChange={this.onStepChange}
-                                                        fromStore={false} />
+                                                        fromStore={false}
+                                                        org={this.props.org}
```
Comment (bobeal) [inline portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/create-or-modify-organization.jsx.js]: I find it weird to have a component with an *organization* and an *org* property.

File: portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/network.jsx.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ var MyNetwork = React.createClass({
         return (
                 <div>
                     <SearchOrganizationModal ref="searchOrgDialog" successHandler={this.openCreateOrgDialog} />
-                    <CreateOrModifyOrganizationModal ref="createOrgDialog" successHandler={this.reload} />
```
Comment (bobeal) [inline portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/network.jsx.js]: Ouch, you broke the « Add an organization » feature !

**Comment by bobeal** [comment]
The issue #260 is about « allowing non admin org members to still edit & save Datacore org info ». But that does not match the content of the commit.

Either an update is needed in the issue, or a new issue is needed, or ...

File: portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/create-or-modify-organization.jsx.js (hunk 1 above)
Comment (MLE-Astek) [inline portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/create-or-modify-organization.jsx.js]: Yes, it's not very well named but it's different data for the same entity. I'll try to find a better way or name.

File: portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/network.jsx.js (hunk 2 above)
Comment (MLE-Astek) [inline portal-parent/ozwillo-portal-front/src/main/resources/public/js/network/network.jsx.js]: Ok, i fix it

**Pull request closed without merging**